sdk/go/common/util/logging/log.go

```diff
@@ -44,8 +44,9 @@ var (
 )
 
 var (
-	rwLock  sync.RWMutex
-	filters []Filter
+	rwLock       sync.RWMutex
+	globalFilter *globalAhoCorasickFilter
+	filters      []Filter
```

Comment:
While we're here, can we make these names a little longer to avoid accidentally referring to them in functions? Especially `filters`. It's too easy to accidentally refer to.

```suggestion
	globalLock sync.RWMutex
	globalACFilter *globalAhoCorasickFilter
	globalFiltersList []Filter
```

FWIW, at `$lastJob` we followed [this guidance](https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_), which technically goes against the convention of "don't put underscores in names" but had a serious significant benefit of separating the namespace of unexported mutable global state from everything else. (I'm not advocating for following that guidance here yet.)